share/commands/relay-reload

```diff
@@ -26,6 +29,9 @@ while getopts "qciIvd" opt; do
     c)
       clear_colors
       ;;
+    p)
+      chgrp rudder /var/rudder/lib/ssl/allnodescerts.pem
```

Comment:
this should only be done if the file exists (because it won't be there on simple relays)

Reply:
note that the reload/start will fail if the file is not there